test_plugin/main.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+const { dlname, dlopen } = Deno;
+
+let localPath: any = import.meta.url.split("/");
+localPath.pop();
+localPath = localPath.join("/");
+
+const dLib = dlopen(env().DL_PATH_TEST_BINDING + "/" + dlname("test_plugin"));
```

Comment:
We should use either "plugin" or "dlopen" but not both.

I think `Deno.openPlugin()` ?

Reply:
`dlname` can be implemented purely in TS.

Reply:
> `dlname` can be implemented purely in TS.

Is it not already?